core/src/mindustry/world/blocks/defense/ShockMine.java

```diff
@@ -46,17 +49,26 @@ public void draw(){
 
         @Override
         public void drawCracks(){
-
+            //no
         }
 
         @Override
         public void unitOn(Unit unit){
             if(enabled && unit.team != team && timer(timerDamage, cooldown)){
-                for(int i = 0; i < tendrils; i++){
-                    Lightning.create(team, lightningColor, damage, x, y, Mathf.random(360f), length);
-                }
+                triggered();
                 damage(tileDamage);
             }
         }
+
+        public void triggered(){
+            for(int i = 0; i < tendrils; i++){
+                Lightning.create(team, lightningColor, damage, x, y, Mathf.random(360f), length);
+            }
+            if(bullet != null){
+                for(int i = 0; i < shots; i++){
+                    bullet.create(this, x, y, Mathf.random(360f));
```

Comment:
would dividing 360 by the amount of bullets not make more sense? 
(equal bullets in all direction, possibly angled towards the initial unit in case of just 1 bullet)